sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.0.0b13 (Unreleased)
```

Comment:
I think you also need to increment the version https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-projects/azure/ai/projects/_version.py#L9